doc/source/reference/fairseq2.data_type.rst

```diff
@@ -8,5 +8,4 @@ fairseq2.data_type
 Functions
 =========
 
-.. autofunction:: current_dtype
```

Comment:
This is a "bug" on my end. Let me fix it in another PR and rebase this PR on latest main afterwards.